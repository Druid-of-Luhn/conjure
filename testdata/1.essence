language Essence 2.0

minimising x

such that
    x = 3
