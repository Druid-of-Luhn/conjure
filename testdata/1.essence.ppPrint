Spec
  { language = "Essence"
  , version = [ 2
              , 0
              ]
  , topLevelBindings = []
  , topLevelWheres = []
  , objective = Just ( Identifier "x" )
  , constraints = [ GenericNode
                      Eq
                      [ Identifier "x"
                      , ValueInteger 3
                      ]
                  ]
  }