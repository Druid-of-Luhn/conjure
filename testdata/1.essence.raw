Spec
  { language = "Essence"
  , version = [ 2
              , 0
              ]
  , topLevelBindings = [ ( Find
                         , "x"
                         , DomainIntegerFromTo (
                             Just ( ValueInteger 1 ) ) (
                             Just ( ValueInteger 9 ) )
                         )
                       ]
  , topLevelWheres = []
  , objective = Just
                  ( Minimising
                  , Identifier "x"
                  )
  , constraints = [ GenericNode
                      Eq
                      [ Identifier "x"
                      , ValueInteger 3
                      ]
                  ]
  }