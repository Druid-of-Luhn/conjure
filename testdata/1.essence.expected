language Essence 2.0

find x : int(1..9)

minimising x

such that
    x = 3
